### PR TITLE
Move pause and freeze metrics from webrtc-provisional-stats.

### LINKED ISSUE #695: add video totalFreezesDuration &totalPausesDuration etc to  standard getStats
video totalFreezesDuration,totalPausesDuration relate call stats are not in standard getStats api call , its very important for stats overall video stall rate of a call , RTC system usually use reported stall rate drive or measure their qos optimization， we known chrome add video stall stats to [webrtc-provisional-stats](https://w3c.github.io/webrtc-provisional-stats/), but developer can't get it via api call.

### PR DESCRIPTION
**Pull request by henbos**
Fixes #695

***
<a href="https://pr-preview.s3.amazonaws.com/henbos/webrtc-stats/pull/701.html" title="Last updated on Oct 3, 2022, 3:07 PM UTC (864509d)">Preview</a> | <a href="https://pr-preview.s3.amazonaws.com/w3c/webrtc-stats/701/cd1ff2f...henbos:864509d.html" title="Last updated on Oct 3, 2022, 3:07 PM UTC (864509d)">Diff</a>

### CODE REVIEW

**Comment by henbos** [comment]
As part of the moved I fixed some types and clarifications to match the existing implementation.

File: webrtc-stats.html [hunk 1]
```diff
@@ -1187,6 +1191,53 @@ <h2>
...
+              <dt>
+                <dfn><code>freezeCount</code></dfn> of type
+                <span class="idlMemberType">unsigned long</span>
+              </dt>
+              <dd>
+                <p>
+                  Count the total number of video freezes experienced by this receiver.
+                  It is a freeze if frame duration, which is time interval between two
+                  consecutively rendered frames, is equal or exceeds
+                  Max(3 * avg_frame_duration_ms, avg_frame_duration_ms + 150), where
```
Comment (vr000m) [inline webrtc-stats.html:1225]: should the freeze duration be smaller than the pause duration?

**Review by vr000m — APPROVED** [review]
minor comments, otherwise LGTM.

**Comment by vr000m** [comment]
Are the freezes and pauses after decoding or within the jitterbuffer?

**Pull request merged**